config/200-clusterrole-namespaced.yaml

```diff
@@ -0,0 +1,48 @@
+# Copyright 2019 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: knative-eventing-namespaced-admin
```

Comment:
Why is this named 'namespaced-admin' if it is a ClusterRole, and therefore not namespaced?

Reply:
Thanks @Harwayne, The specific name was cargo-culted from the serving PR, only changing what was needed for eventing specifically in this case to remain consistent.

Reply:
Not a fan of the name, but consistency is likely more important.